agent/php_execute.c

```diff
@@ -1966,20 +1967,40 @@ static void nr_php_observer_attempt_call_cufa_handler(NR_EXECUTE_PROTO) {
    * cause additional performance overhead, this should be considered a last
    * resort.
    */
-  const zend_op* prev_opline = execute_data->prev_execute_data->opline - 1;
+
+  /* 
+   * We cannot safely access the opline of internal functions, and we only
+   * want to instrument cufa calls from user functions anyway
```

Comment:
```suggestion
   * Because this code executes in the context of zend_execute and not in the
   * context of VM, we need to ensure we're dealing with a user function. We
   * cannot safely access the opline of internal functions, and we only want
   * to instrument cufa calls from user functions anyway.
```